test/quantization/fx/test_numeric_suite_fx.py

```diff
@@ -2060,6 +2070,223 @@ def forward(self, x):
+
+        results = extract_results_n_shadows_model(msq)
+        print_comparisons_n_shadows_model(results)
+
+    def test_linear_mod(self):
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.fc1 = nn.Linear(2, 2)
+
+            def forward(self, x):
+                x = self.fc1(x)
+                return x
+
+        m = M().eval()
+        example_input = (torch.randn(2, 2),)
+
+        qconfig_mappings = [
+            QConfigMapping().set_global(torch.quantization.default_qconfig),
+        ]
+        self._test_impl(m, example_input, qconfig_mappings)
+
+    def test_linear_relu_mod(self):
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.fc1 = nn.Linear(2, 2)
+                self.fc2 = nn.Linear(2, 2)
+                self.relu = nn.ReLU()
+
+            def forward(self, x):
+                x = self.fc1(x)
+                x = self.fc2(x)
+                x = self.relu(x)
+                return x
+
+        m = M().eval()
+        example_input = (torch.randn(2, 2),)
+
+        qconfig_mappings = [
+            QConfigMapping().set_global(torch.quantization.default_qconfig),
+            QConfigMapping().set_global(torch.quantization.default_dynamic_qconfig),
+        ]
+        self._test_impl(m, example_input, qconfig_mappings)
+
+    def test_conv_bn_relu_mod(self):
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = nn.Conv2d(1, 1, 1)
+                self.bn = nn.BatchNorm2d(1)
+                self.relu = nn.ReLU()
+
+            def forward(self, x):
+                x = self.conv(x)
+                x = self.bn(x)
+                x = self.relu(x)
+                return x
+
+        m = M().eval()
+        example_input = (torch.randn(32, 1, 16, 16),)
+        qconfig_mappings = [
+            QConfigMapping().set_global(torch.quantization.default_qconfig),
+            QConfigMapping().set_global(torch.quantization.default_per_channel_qconfig),
+        ]
+        self._test_impl(m, example_input, qconfig_mappings)
+
+    def test_functions(self):
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.w1 = nn.Parameter(torch.randn(2, 2))
+                self.b1 = nn.Parameter(torch.zeros(2))
+                torch.nn.init.kaiming_uniform_(self.w1, a=math.sqrt(5))
+
+            def forward(self, x):
+                x = F.sigmoid(x)
+                x = F.linear(x, self.w1, self.b1)
+                x = F.linear(x, self.w1[:], self.b1)
+                x = F.relu(x)
+                x = x + x
+                x = torch.cat([x])
+                x = torch.cat((x,))
+                x = torch.cat(tensors=[x])
+                # TODO(future PR): enable layernorm
+                # blocked on FX graph mode quant not inserting observer for
```

Comment:
is this fixed now? with https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/backend_config/_common_operator_config_utils.py#L355